src/services/services.ts

```diff
@@ -4578,13 +4579,22 @@ namespace ts {
                 return false;
             }
 
+           function tryAddVariableSignature(declarations: Declaration[], location: Node, symbolKind: string, symbolName: string, containerName: string, result: DefinitionInfo[]) {
+                forEach(declarations, declaration => {
+                    if (declaration.kind === ts.SyntaxKind.VariableDeclaration) {
+                        return tryAddSignature(symbol.declarations, /*selectConstructors*/ true, symbolKind, symbolName, containerName, result);

```

Comment:
No, I don't. But, if this parameter chenge `true` to `false`, **goToDefinitionShorthandProperty01.ts** test case fail.